Expensify/App
# [No QA] Fix directory error on node run web

### Details
This makes config/webpack/CustomVersionFilePlugin.js ensure all of version.js's parent directories exist before creating it.

### Fixed Issues
$ https://github.com/Expensify/App/issues/7011

### Tests

1. Make a fresh copy of the repo
2. Install node & npm: `brew install node`
3. Install watchman: `brew install watchman`
4. Install dependencies: `npm install`
5. Run `npm run web`


### QA Steps
I think simply deploying this to staging should validate it worked if staging has a fresh copy of the repo each time.


### Tested On


### Screenshots
N/A, this is dev bug

#### Web
N/A

#### Mobile Web
N/A

#### Desktop
N/A

#### iOS
N/A

#### Android
N/A
